apps/sms/tools/update_nga_libs.sh

```diff
@@ -12,7 +12,7 @@ bower_path="bower_components"
 
```

Comment:
If we really want to create our own version of minimized/split "bridge", I think it's not that difficult to do it in our build script (or even use npm to setup the lib like threads did. We can output ES6 in our codebase ;) but we might need to ask why Wilson chose ES5 instead). But yes it's easier it we can "bridge" lib naturally :)